x-pack/legacy/plugins/file_upload/public/components/json_upload_and_parse.js

```diff
@@ -86,7 +94,14 @@ export class JsonUploadAndParse extends Component {
     this._setSelectedType(this.state);
     this._setIndexReady({ ...this.state, ...this.props });
     this._indexData({ ...this.state, ...this.props });
-    if (this.props.isIndexingTriggered && !this.state.showImportProgress) {
+    if (!this._isMounted) {
+      return;
+    }
+    if (
+      this.props.isIndexingTriggered &&
+      !this.state.showImportProgress &&
+      this._isMounted
```

Comment:
No need to check for `this._isMounted`, function would have already returned if `this._isMounted` is false at this point